src/Sphere.jl

```diff
@@ -36,25 +36,25 @@ end
 zero_tangent_vector(S::Sphere, x) = zero(x)
 zero_tangent_vector!(S::Sphere, v, x) = (v .= zero(x))
 
-function is_manifold_point(S::Sphere,x)
+function is_manifold_point(S::Sphere,x, local_isapprox=Base.isapprox)
```

Comment:
It feels a bit clunky passing a whole approx function. I was thinking something more like.

```julia
function is_manifold_point(S::Sphere, x; kwargs...)
   ...
   if isapprox(norm(x), 1; kwargs...)
      ...
   end
end
```
This also makes it easier to change the tolerances. Instead of needing to call something like `is_manifold_point(m, x, x->isapprox(x; atol=1e-3))`, you'd call `is_manifold_point(m, x; atol=1e-3)`.

Reply:
Thanks for the good idea, will add that today.